src/net_processing.cpp

```diff
@@ -3782,7 +3781,7 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
 
         if (vAddr.size() > MAX_ADDR_TO_SEND)
         {
-            Misbehaving(*peer, 20, strprintf("%s message size = %u", msg_type, vAddr.size()));
+            Misbehaving(*peer, 100, strprintf("%s message size = %u", msg_type, vAddr.size()));
```

Comment:
I believe this limit was introduced a decade ago in #5419, at least on the sender side.